### ISSUE TITLE
FALCON: Division and Batch Normalization

### ISSUE THREAD

**Issue by abogaziah** [OpenMined/PySyft#3972]
## Feature Description
Algorithm 6 is used to compute the value of a/b where a,b are secret shared
Batch-norm is another important component of neural network architectures. They improve the convergence as well
as help automate the training process. Algorithm 7 describes
the protocol to compute batch-norm

## Additional Context
Division and Batch Normalization is described in section 3.7 in [FALCON paper](https://arxiv.org/pdf/2004.02229v1.pdf)
